docs/getting-started/install-kubernetes.md

```diff
@@ -17,13 +17,15 @@ description: >
   - [Deploying into a custom namespace](#deploying-into-a-custom-namespace)
   - [Hook Mode](#hook-mode)
   - [Deploying with an AppArmor profile](#deploying-with-an-apparmor-profile)
+  - [Helm Chart Installation](#helm-chart-installation)
   - [Specific Information for Different Platforms](#specific-information-for-different-platforms)
     - [Minikube](#minikube)
 - [Uninstalling from the cluster](#uninstalling-from-the-cluster)
 - [Version skew policy](#version-skew-policy)
 - [Installing `ig`](#installing-ig)
   - [Install a specific release](#install-a-specific-release-1)
   - [Compile from source](#compile-from-source-1)
+  - [Using ig without installation](#using-ig-without-installation)
```

Comment:
I wonder if it should be in its own top section and cover both running ig without installation on 1. Linux hosts and 2. Kubernetes clusters (and just link to the appropriate part of ig.md).

Reply:
Agreed, it looked bit off here. I added a new section "Quick Start" to make it clear. Please feel free to have another look! :)